components/config/event.js

```diff
@@ -0,0 +1,19 @@
+
+const EVENT_DATA = [
+  {
+    "kind": "calendar#event",
+    "id": "jasjkfbakjfbjgbebzbcbau",
+    "summary": "GBM 1",
+    "description": "Our first General Body Meeting of the Spring Semester. Come out for free pizza and to learn about upcoming ACE events!",
+    "location": "TBA",
+    "datetime": "1/15 6:30PM"
+  },
+  {
+    "summary": "Ultimate Software Event",
+    "description": "bruhhh",
```

Comment:
Junk data

Reply:
I edited the description, date and location for the ultimate event